bzt/modules/blazemeter.py

```diff
@@ -392,7 +392,7 @@ def monitoring_data(self, data):
     def __send_monitoring(self):
         src_name = platform.node()
         data = self.monitoring_buffer.get_monitoring_json(self._session)
-        self._session.send_monitoring_data(src_name, data)
+        self._session.send_monitoring_data(id(self.engine), data)
```

Comment:
The engine id is the same as in `worker_index` from `__upload_artifacts`, with default of zero